lib/runner.js

```diff
@@ -113,6 +120,16 @@ class Runner extends Emittery {
 					throw new TypeError('Expected an implementation. Use `test.todo()` for tests without an implementation.');
 				}
 
+				if (metadata.skipped && this.updateSnapshots) {
+					this.updateSnapshots = null;
+					this.errorUpdatingSnapshots = new SnapshotError('Tests can not be skipped when snapshots should be updated.');
+				}
```

Comment:
In issue #1784 skip was one condition, were updating snapshots should be blocked. For me it sounds logical, because you can also skip finished tests. 
Instead of skip, you can also comment your test out 😊.

Reply:
> In issue #1784 skip was one condition, were updating snapshots should be blocked. For me it sounds logical, because you can also skip finished tests.

Yea let's just go with this. We can always undo 😄